src/application/components/applicationFormSubsection.tsx

```diff
@@ -107,6 +109,8 @@ const ApplicationFormSubsectionFields = connect(
       props.identifier,
       props.formName
     ),
+    getValue: (fieldIdentifier: string): string =>
```

Comment:
Defaults are always strings, but the actual field value in the general case is not guaranteed to be (checkbox fields are arrays and single-checkbox fields without choices are booleans for example). I think `FieldValue` in `application/types.ts` is somewhat representative of a possible type, though it appears to be unused for some reason.

Reply:
The return type of this (and `getFieldValue` used by this) weren't yet changed in the last revision.